finance/bitcoin/Portfile

```diff
@@ -103,7 +103,6 @@ variant daemon description {Build and install the daemon} {
 
 variant gui description {Build the Qt5 GUI} {
     PortGroup qt5 1.0
-    PortGroup cxx11 1.1
```

Comment:
You didn't explain in your pull request why you removed this...

Reply:
Keeping that line and running `port lint bitcoin` returns:

```
--->  Verifying Portfile for bitcoin
Error: Line 106 repeats inclusion of PortGroup cxx11
--->  1 errors and 0 warnings found.
```

Should that error be fixed in another way?

Reply:
No, you seem to have fixed it correctly, but you should explain that you removed the line to stop a lint error in the expanded section of the commit message and in your pull request. First comes the summary line, then comes a more effusive explanation in the paragraph following. That way someone knows why you did what you did.

Reply:
For example:

```
bitcoin: update to 0.16.1, fix lint error

The cxx11 portgroup was incorrectly included twice in the portfile.
```

Reply:
Thank you for letting me know. I will include more explanations in the commits from now on.